student.rb

```diff
@@ -1,11 +1,16 @@
 require './person'
 
 class Studentes < Person
-  def intializate(classroom, age, name = 'Unknown', parent_permission: true)
+  def initialize(classroom, age, name = 'Unknown', parent_permission: true)
```

Comment:
- [ ] As suggested already, kindly add `attr_reader :classroom` to provide a getter method for accessing a classroom a student belong to.